src/pybind/mgr/dashboard/settings.py

```diff
@@ -118,12 +137,17 @@ def filter_attr(member):
 def handle_option_command(cmd):
     if cmd['prefix'] not in _OPTIONS_COMMAND_MAP:
         return (-errno.ENOSYS, '', "Command not found '{}'".format(cmd['prefix']))
-
     opt = _OPTIONS_COMMAND_MAP[cmd['prefix']]
-    if not opt['type']:
-        # get option
-        return 0, str(getattr(Settings, opt['name'])), ''
 
-    # set option
-    setattr(Settings, opt['name'], opt['type'](cmd['value']))
-    return 0, 'Option {} updated'.format(opt['name']), ''
+    if cmd['prefix'].startswith('dashboard reset'):
+        delattr(Settings, opt['name'])
+        return 0, 'Option {} reset to default value "{}"'.format(
+            opt['name'], getattr(Settings, opt['name'])), ''
+    elif cmd['prefix'].startswith('dashboard get'):
+        return 0, str(getattr(Settings, opt['name'])), ''
+    elif cmd['prefix'].startswith('dashboard set'):
+        value = opt['type'](cmd['value'])
+        if opt['type'] == bool and cmd['value'].lower() == 'false':
```

Comment:
Afaik we already have a function for converting strings to bools somewhere.

Reply:
Where? I couldn't find it. And does that mean you'd suggest to use a function to do `.lower() == 'false'`? Because that's really everything what's required here.

Reply:
I wouldn't mind in not using this function, but this would be an opportunity to be consistent across the dashboard.

Reply:
I tend to not implement it as it doesn't solve any problem but'd introduce behavior we likely won't document nor use, because it's not required to have those options to set a true value. But if anyone things we should really do that, I'd just do it.